samcli/lib/docs/documentation_links.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
do we need to keep the keys in this doc in any specific order, or it does not matter ?

Reply:
Shouldn't matter